Tools/CISupport/ews-build/factories.py

```diff
@@ -375,3 +376,10 @@ def __init__(self, platform, **kwargs):
         self.addStep(UpdatePullRequest())
         self.addStep(PushCommitToWebKitRepo())
         self.addStep(SetBuildSummary())
+
+
+class SafeMergeQueueFactory(factory.BuildFactory):
+    def __init__(self, platform, configuration=None, architectures=None, additionalArguments=None, **kwargs):
+        super().__init__()
+        for project in GITHUB_PROJECTS:
+            self.addStep(RetrievePRDataFromLabel(project, label='\"safe-merge-queue\"'))
```

Comment:
Nit: I don't think we should be quoting the label name here. And the label name should probably be a variable (as our other label names names are). I do really like the fact that this label is passed as an argument to `RetrievePRDataFromLabel`, so let's be suer to keep that!